app/templates/results.html

```diff
@@ -87,6 +86,15 @@ <h4>License:</h4>
       </div>
       <button class="show_more_button" >Show more</button>
       <button class="show_less_button" >Show less</button>
+
```

Comment:
this will work, but the drawback is that the buttons may flash in and out of view if the CSS is slow to load.

Instead, try putting the buttons inside a `{% if nojs is not defined %}`

How does it look without them? Does the whole result show up as expected?

Reply:
That makes sense.
Yes it shows the whole results as expected.